README.md

```diff
@@ -81,6 +81,27 @@ puts response.status_code
 puts response.body
 puts response.headers
 ```
+## Manage Account Settings
```

Comment:
Cool makes sense, any preference on how I link to it from the main README? I didnt see a link to the mail helper there currently